backend/API_standards.md

```diff
@@ -161,7 +162,8 @@ Not Found example:
   "errors": [
     {
       "title": "Resource not found",
-      "message": "User not found, please check your identifier"
+      "message": "User not found, please check your identifier",
+      "timestamp": "2018-03-12T18:30:00.SSSZ"
```

Comment:
We currently have timestamp on backend and Papertrail logs, can you tell a bit more about the benefits of having this for the API consumer?